recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "docrepr" %}
-{% set version = "0.1.1" %}
+{% set version = "0.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d7b27b0105d19267be74ff8048f213b8968ff366ded45357d112ef02e109f715
+  sha256: 3ad62ecdaa0b7ab90dcd9c29abc98cae7d61d09339f18396d4ca5b8ab41c8b90
 
 build:
   noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
```

Comment:
We probably need to update the sphinx version

Reply:
Yeah; the pin we changed it to upstream now is `sphinx>1.1`, though realistically we could pin it higher since 1.1 is extremely old.

Reply:
Also, the host requirements should be updated to include `setuptools>42`; right now its relying on it implicitly.

Reply:
Thanks! I added `sphinx>1.1`, `setuptools>42`